tests/linearGaussian_npse_test.py

```diff
@@ -272,27 +274,25 @@ def test_npse_iid_inference(
 
 
 @pytest.mark.slow
-def test_npse_map():
-    num_dim = 2
-    x_o = zeros(num_dim)
-    num_simulations = 3000
-
-    # likelihood_mean will be likelihood_shift+theta
-    likelihood_shift = -1.0 * ones(num_dim)
-    likelihood_cov = 0.3 * eye(num_dim)
+@pytest.mark.parametrize(
+    "npse_trained_model",
+    [
+        ("vp", "gaussian"),
```

Comment:
👍 
probably fine to remove the comma here to reduce the formatting to fewer lines.